src/main/webapp/style.css

```diff
@@ -98,6 +98,78 @@ input[type=submit]:hover {
   text-align: center;
 }
 
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
```

Comment:
When I ask an all-states query, the first thing that happens is that the title appears, the toggle button appears, and the loading text appears. The second thing that happens is that the map replaces the loading text and the toggle button disappears. If the loading takes a few seconds, then it's really noticeable that the toggle button appears only to disappear again.
To avoid this, maybe we should have the toggle be default hidden, and it only gets made visible if it's a specific-state query, rather than vice versa.

Reply:
Made toggle hidden by default!